src/EventPayload.ts

```diff
@@ -8,6 +8,8 @@ import { Action } from './Action';
 export interface EventPayload {
   /** The user action which caused the event, e.g. ADD_TO_CART or THUMBS_UP  */
   action: Action;
+  /** The authorization token for the request. Formatted like KEY <api-key> or Bearer <bearer-token> */
```

Comment:
nit: add a note saying that this will be set up using the auth token passed into the config